atomistic.ttl

```diff
@@ -70,8 +70,22 @@ email: francesca.l.bleken@sintef.no"""@en ,
 
 
 #################################################################
-#    Classes
+#    Classes
 #################################################################
+
+###  https://w3id.org/emmo/domain/atomistic#AtomisticComputationSetting
```

Comment:
For consistency I think we should change this to a numerical IRI. With DLite, you can easily generate new UUIDs by running the `dlite-getuuid` tool without arguments.